src/dotnet/Program.cs

```diff
@@ -36,6 +37,7 @@ public class Program
             ["build"] = BuildCommand.Run,
             ["clean"] = CleanCommand.Run,
             ["help"] = HelpCommand.Run,
+            ["list"] = (new ListCommand()).Run,
```

Comment:
look at `dotnet add` and `dotnet remove` here and also at src/dotnet/DispatchCommand.cs - this is to prevent logic repetition